src/handlers/schedule-checkin.ts

```diff
@@ -146,7 +147,7 @@ async function findFirstLegDate(reservation: Reservation.Reservation) {
     return;
   }
 
-  return Luxon.DateTime.min(...validLegs).toJSDate();
+  return validLegs;
```

Comment:
I'd prefer to return native JS Dates. Passing a library's dates between functions makes it harder to swap libraries if necessary.

Reply:
That makes sense!

I initially did this because I saw we'd have to convert back and forth, but the ability to swap libraries definitely is beneficial.

Reply:
It's a little annoying but I generally think it's worth it. Thanks for updating!